### ISSUE TITLE
Broken

### ISSUE THREAD

**Issue by maplepy** [WayneYellow/Deep-Rock-Galactic-Save-Editor#3]
will reset your profile to lvl 1 and give you the starting tutorial

**Comment by WayneYellow** [comment]
Yeah, the official seems to have changed something, I'm still trying to figure it out

**Comment by maplepy** [comment]
If you want to take a look at https://github.com/trylogout/drg-editor, this one works great
